.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -51,12 +51,6 @@ jobs:
         run: |
           ./.ci/close-create-milestone.sh
 
-      - name: Upload '-all' jar
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        run: |
-          ./.ci/upload-all-jar.sh ${{ inputs.version }}
-
```

Comment:
After merging of PR, https://github.com/checkstyle/checkstyle/wiki/Semi-automated-release has to be updated.

Reply:
I will open another PR for this actually.